pkg/archive/archive.go

```diff
@@ -420,16 +422,16 @@ func FileInfoHeader(name string, fi os.FileInfo, link string) (*tar.Header, erro
 // ReadSecurityXattrToTarHeader reads security.capability, security,image
 // xattrs from filesystem to a tar header
 func ReadSecurityXattrToTarHeader(path string, hdr *tar.Header) error {
```

Comment:
This, and `ReadUserXattrToTarHeader`, now change how they output data. That _should_ be fine locally, but it might trip up callers that make assumptions (like `overlayWhiteoutConverter` does).

I’d prefer those two functions to be made private when we are changing them; I don’t expect there to be external users.